src/bundle/Resources/views/themes/admin/content_type/macros.html.twig

```diff
@@ -13,4 +13,19 @@
             <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#edit"></use>
         </svg>
     </a>
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro content_type_copy(content_type, content_type_group, class_name) %}
+    {% set copy_url = path('ezplatform.content_type.copy', {
+        'contentTypeGroupId': content_type_group.id,
+        'contentTypeId': content_type.id
+    }) %}
+
+    <a title="{{ 'content_type.action.copy'|trans|desc('Copy') }}"
+       href="{{ copy_url }}"
+       {% if class_name %}class="{{ class_name }}"{% endif %}>
+        <svg class="ez-icon ez-icon-copy">
```

Comment:
I would suggest to add also `ez-icon--secondary`, given that it is not a primary action. That way it would look more consistent with other tables in the app.